src/pfe/portal/modules/utils/metricsStatusChecker.js

```diff
@@ -42,7 +41,6 @@ async function isMetricsAvailable(projectPath, projectLanguage) {
   if (await fs.pathExists(pathOfFileToCheck)) {
     return doesMetricsPackageExist(pathOfFileToCheck, projectLanguage)
   }
-  throw new MetricsStatusError("BUILD_FILE_MISSING", `Cannot find project build-file (${fileToCheck})`);
 }
```

Comment:
Now that `retrieveProject` calls this function, this function gets called during project `upload` (for every file). We upload files before the `package.json` (or java equivalent) is uploaded, triggering this error. 

Prior to this PR, the only effect of this error was to make PFE return status 404 when we cannot find the build file (see https://github.com/eclipse/codewind/pull/2043/files#diff-e92faeededd9ee751c7dddf842d0bb86L81-L85). By removing it, we will now return status 500. I'm not sure if anyone was using the 404, but if needed we can add it back in, as long as it does not make `upload` error!

Reply:
I've since changed the design to not need this change